### PR TITLE
chore: issue templates for bugs and features

### PR DESCRIPTION
**Pull request by ZyanKLee**
This is just a suggestion on how these could look - your input is very much welcome and even requested

### CODE REVIEW

**Comment by ZyanKLee** [comment]
besides: this is the first time I set templates up for github - so I'm not actually sure this will work as expected without further settings to be changed by @MiczFlor

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+## Hardware
+
+### RaspberryPi version
+
+i.e. `3 B+`
+
+### RFID Reader
+
```
Comment (s-martin) [inline .github/ISSUE_TEMPLATE/bug_template.md]: Comment how the version can be obtained would be helpful.

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+i.e. `16c0:27db HXGCoLtd Keyboard`
+
+### Soundcard
+
+i.e. `0d8c:0014 C-Media Electronics, Inc. Audio Adapter (Unitek Y-247A)`
```
Comment (s-martin) [inline .github/ISSUE_TEMPLATE/bug_template.md]: See above

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+### Other notable hardware
+
+i.e.:
+```
+GPIO Buttons:
```
Comment (s-martin) [inline .github/ISSUE_TEMPLATE/bug_template.md]: Comment like "Please paste content from /home/pi/RPi-Jukebox-RFID/scripts/gpio_buttons.py here" would be helpful

File: .github/ISSUE_TEMPLATE/bug_template.md (hunk 1 above)
Comment (ZyanKLee) [reply .github/ISSUE_TEMPLATE/bug_template.md]: good idea.

File: .github/ISSUE_TEMPLATE/bug_template.md (hunk 3 above)
Comment (ZyanKLee) [reply .github/ISSUE_TEMPLATE/bug_template.md]: good idea.

File: .github/ISSUE_TEMPLATE/bug_template.md (hunk 2 above)
Comment (ZyanKLee) [reply .github/ISSUE_TEMPLATE/bug_template.md]: good idea.

**Comment by MiczFlor** [comment]
Let's give it a try :)

**Comment by s-martin** [comment]
Just found that there is a script to gather information for bug fixing ( see https://github.com/MiczFlor/RPi-Jukebox-RFID#reporting-bugs).

Probably makes sense to include that into the bugs template

**Pull request merged**